Streamline navigation bar, tweak a couple of presentation things

anaulin opened the pull request:
As discussed at the last meeting, here is a PR with a proposal to tighten up the navigation bar. One goal is to make space for other navigation bar items we might want to add (e.g. "editorial direction").

Changes in this PR:
* Removed the link to GitHub from the navigation area altogether. We already have a link to our GitHub in the site footer, and it seems less relevant anyway.
* Replaced the word "twitter" with an icon. Now it takes more space, but perhaps makes Twitter oddly prominent.
* Removed the word "search" in the navbar with an icon.
* Removed extra mention about the site being "open source" from footer, to make footer smaller. (In my mind the size of the footer doesn't matter as much as other things, though). Tightened it by putting emoji and text on same line.
* Made link color a darker, less saturated red. Previous bright red shows on link hover.

Screenshots after the changes:

New navbar look:
![Screen Shot 2020-08-08 at 10 12 55 AM](https://user-images.githubusercontent.com/6729309/89716259-51c25700-d960-11ea-8fe1-7468ac8358e3.png)

New footer:
![Screen Shot 2020-08-08 at 10 18 13 AM](https://user-images.githubusercontent.com/6729309/89716279-7f0f0500-d960-11ea-87ba-30ca09e8e819.png)


New archive look, with less bright links:
![Screen Shot 2020-08-08 at 10 13 03 AM](https://user-images.githubusercontent.com/6729309/89716267-60a90980-d960-11ea-88ab-d4f410b48d11.png)

stationaery commented:
On all counts, the navbar and icon and archive layout, L~G~T~M~! Nice work, Ana 😊

I might add that the footer could be streamlined, too - either nipping the "This site is open source." text or "or open a GitHub issue" text.

I also think borders around images could get bipped, too.

anaulin commented:
Made the footer a little smaller. Looks like this:

![Screen Shot 2020-08-07 at 4 05 06 PM](https://user-images.githubusercontent.com/6729309/89695183-e9c03200-d8c7-11ea-8100-4ef687b6217d.png)

anaulin commented:
Removed image border, now images look like this:

![Screen Shot 2020-08-07 at 5 34 37 PM](https://user-images.githubusercontent.com/6729309/89698135-5d683c00-d8d4-11ea-95d4-0f29358d715f.png)

anaulin commented:
@stationaery removed image borders and made footer smaller. See screenshots. How's it look?

anaulin commented:
I played a little bit with making the red color of our links a little bit less aggressive (I think there is too much bright red in our "news" section).

Here are some screenshots of an experiment replacing the current color (#e3252b) with a less saturated and darker one (#c7484c):

![Screen Shot 2020-08-07 at 5 48 59 PM](https://user-images.githubusercontent.com/6729309/89698540-af11c600-d8d6-11ea-812e-c01e617f4eff.png)

![Screen Shot 2020-08-07 at 5 48 39 PM](https://user-images.githubusercontent.com/6729309/89698554-bd5fe200-d8d6-11ea-9f2a-c52fbe827b98.png)

This is not included in this PR, for now, but could be. It feels like a slight improvement to me, but I am not sure if it is too subtle to matter. Perhaps we should consider moving to a different link color altogether (or e.g. underline instead of a different color).

jessesquires commented on search.md:
```diff
@@ -1,8 +0,0 @@
-
-<p class="text-muted">Search everything on this site.</p>
-
-{% include search_box.html %}
```
i think this is still worth keeping, even if not prominently linked in the nav bar.

but, not a big deal

anaulin replied:
Brought it back, linked in bar with icon. And removed search from archive, since now we have this back.

jessesquires commented on archive.md:
```diff
@@ -3,6 +3,12 @@ layout: standalone
+    Search the archives or browse in reverse chronological order below.
+
+    {% include search_box.html %}
+</p>
```
i don't think we need the `<p>` tags here, do we?

if we need to adjust margin/padding, we can use something like `<div class="mt-2 mb-2">` instead 

(for margin-top and margin-bottom, can specify 1-4)

jessesquires commented on _includes/image.html:
```diff
@@ -18,7 +18,7 @@
               <a href="{{ source_link }}" target="_blank" rel="noreferrer">
             {% endif %}
-            <img class="img-thumbnail img-fluid"
+            <img class="img-thumbnail img-fluid border-0"
```
I'm strongly in favor of keeping the border styling on images. 😊 

I think it looks much better. 😄

anaulin replied:
Brought border back.

jessesquires commented on _includes/footer.html:
```diff
@@ -7,10 +7,9 @@
-        ✊💛
+        ✊ 💛
         <br/>
         A <a href="https://techworkerscoalition.org">Tech Workers Coalition</a> project.
```
should we just put this on the same line as the emoji now?

that might look better.

anaulin replied:
Done.

jessesquires commented on _includes/header.html:
```diff
@@ -19,13 +19,7 @@
-            </li>
-            <li class="nav-item mr-2">
-                <a class="nav-link" href="https://twitter.com/{{ site.author.twitter }}">twitter</a>
+                <a class="nav-link" href="https://twitter.com/{{ site.author.twitter }}"><i class="fab fa-twitter"></i></a>
```
I'd be down for a search icon in the nav bar -- but I don't feel strongly about that.

anaulin replied:
Done.

jessesquires commented:
> I played a little bit with making the red color of our links a little bit less aggressive (I think there is too much bright red in our "news" section).

I like the less saturated red 💯 

Just had an idea: what if we just swap the current `link` and `hover` colors? so links are darker/desaturated and on hover, they use the brighter red

anaulin commented:
Thanks for all the feedback, @jessesquires.

I've taken your idea about the links being brighter on hover, and added that to this PR (together with replacing the "dark" color with the slightly more muted darker red I had in my screenshot above). I've also tightened the footer, brought back the image border, and the standalone search page, linked via icon in navbar.

Full page screenshot with new changes (see also updated screencaps in PR description):
![newsletter-full-page](https://user-images.githubusercontent.com/6729309/89716221-1758ba00-d960-11ea-98da-08f51ae42bd7.png)

stationaery commented:
One tweak: since our newsletter has an extremely vertical orientation, I
think that the ✊💛 icons look much nicer spacing out and dividing the two
lines of the footer. It also makes the lines above and below easier to
read. Can we change it back to that?


On Sat, Aug 8, 2020 at 11:14 AM Jesse Squires <notifications@github.com>
wrote:

> *@jessesquires* approved this pull request.
>
> looks awesome! 🎉
>
> thanks for doing this @anaulin <https://github.com/anaulin> ! 🙌
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/techworkersco/techworkersco.github.io/pull/112#pullrequestreview-463818912>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACES7PMA4CRLBU2PHEVUVSTR7WIZTANCNFSM4PX4V5KQ>
> .
>

anaulin commented:
Ok. I put the emojis on their own line, at the bottom of the footer, which now looks like:

![Screen Shot 2020-08-08 at 12 13 15 PM](https://user-images.githubusercontent.com/6729309/89718007-ad94dc00-d970-11ea-9513-2c222b6d7439.png)

stationaery commented:
😻💛

On Sat, Aug 8, 2020 at 12:14 PM Ana Ulin <notifications@github.com> wrote:

> Ok. I put the emojis on their own line, at the bottom of the footer, which
> now looks like:
>
> [image: Screen Shot 2020-08-08 at 12 13 15 PM]
> <https://user-images.githubusercontent.com/6729309/89718007-ad94dc00-d970-11ea-9513-2c222b6d7439.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/techworkersco/techworkersco.github.io/pull/112#issuecomment-670963821>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACES7PIB7CIAKFT7EPQY66TR7WPZLANCNFSM4PX4V5KQ>
> .
>

anaulin commented:
I'm a gonna merge this as it stands now. If we come up with more tweaks to do, I am happy to do those as a follow-up PR.
